app/src/main/python/gogdl/imports.py

```
@@ -0,0 +1,130 @@
+        else:
+            installed_language = "en-US"
+        if build_id_file:
+            f = open(build_id_file, "r")
+            build = json.loads(f.read())
+            f.close()
+            build_id = build.get("buildId")
+
+        version_name = build_id
+        if build_id and platform != "linux":
+            # Get version name
+            builds_res = requests.get(
```

Ruff (0.12.2)

Probable use of `requests` call without timeout

(S113)